### DISCUSSION TITLE
How to handle symlinks for a project?

### DISCUSSION THREAD

**Discussion by amitjindal** [copier-org/copier#414] [Q&A] [answered]
Hi, I am in a situation where I need to create a project that has several symlinks.
How do I handle that? When I ran copier it made copies of the files and did not preserve symlinks.

Thanks
Amit

**Comment by yajo** [comment, marked as answer]
There's no specific code for this, so I guess it's treated like normal files. 

You could create the symlinks using tasks.

**Reply by amitjindal** [reply]
Okay, thanks a lot @Yajo . That's what I ended up doing.
